CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `actionMenu` prop to `ChatMessage` component @layershifter ([#811](https://github.com/stardust-ui/react/pull/811))
 - Add `rtl` field in the `SvgIconFuncArg`, and used it in Teams theme's number-list icon ([#851](https://github.com/stardust-ui/react/pull/851))
 - Add single search flavor for `Dropdown` component @Bugaa92 ([#839](https://github.com/stardust-ui/react/pull/839))
+- add `black` and `white` options for the `color` prop in the `Label` component @mnajdova ([#855](https://github.com/stardust-ui/react/pull/855))
```

Comment:
nit: lets, please, capitalise this sentence, for the sake of consistency

Reply:
Done.